.gitignore

```diff
@@ -1,5 +1,7 @@
 __pycache__
 
+eeg_notebooks.egg-info/
```

Comment:
This could be *.egg-info, but just nitpicking :)

```suggestion
*.egg-info/
```

Reply:
Looked up about Python "eggs" and made this change!